src/main/java/com/redhat/cloud/notifications/routers/UserConfigService.java

```diff
@@ -136,14 +192,59 @@
         });
     }
 
+    private Uni<SettingsValues> getSettingsValueForUser(String account, String username, String bundleName) {
+        return Uni.createFrom().deferred(() -> {
+            final SettingsValues values = new SettingsValues();
+
+            return bundleResources.getBundles()
+                    // Todo: Create a method that gets the bundle by name
+                    .filter(bundle -> bundle.getName().equals(bundleName))
+                    .onItem().transformToMulti(bundle -> {
+                        BundleSettingsValue bundleSettingsValue = new BundleSettingsValue();
+                        bundleSettingsValue.name = bundle.getDisplay_name();
+                        values.bundles.put(bundle.getName(), bundleSettingsValue);
+                        return applicationResources.getApplications(bundle.getName())
+                                .onItem().transformToMulti(application -> {
+                                    ApplicationSettingsValue applicationSettingsValue = new ApplicationSettingsValue();
+                                    applicationSettingsValue.name = application.getDisplay_name();
+                                    values.bundles.get(bundle.getName()).applications.put(application.getName(), applicationSettingsValue);
+                                    return Multi.createFrom().items(EmailSubscriptionType.values())
+                                            .onItem().transformToMulti(emailSubscriptionType -> {
+                                                values.bundles.get(bundle.getName()).applications.get(application.getName()).notifications.put(emailSubscriptionType, false);
+                                                return Multi.createFrom().empty();
+                                            }).concatenate();
+                                }).concatenate();
+                    }).concatenate().collectItems().asList().onItem().transformToMulti(objects -> emailSubscriptionResources.getEmailSubscriptionsForUser(account, username))
+                    .onItem().transform(emailSubscription -> {
+                        values.bundles.get(emailSubscription.getBundle()).applications.get(emailSubscription.getApplication()).notifications.put(emailSubscription.getType(), true);
+                        return emailSubscription;
+                    }).collectItems().asList().onItem().transform(emailSubscriptions -> values);
+        });
+    }
+
+    // Todo: Remove after all settings are in place for notifications userpreferences section
+    private Uni<String> oldSettingsValuesToJsonForm(Uni<SettingsValues> settingsValuesUni) {
+        return settingsValuesUni.onItem().transform(settingsValues -> OldSettingsValueJsonForm.fromSettingsValue(settingsValues))
+                .onItem().transform(settingsValueJsonForm -> {
+                    try {
+                        return mapper.writeValueAsString(settingsValueJsonForm);
```

Comment:
We should inject and use the `ObjectMapper` produced by Quarkus there, but this is not new so it can be fixed later.